tensorboard/plugins/interactive_inference/README.md

```diff
@@ -292,6 +302,63 @@ for all options you can provide, including how to specify other model types
+model to Cloud AI Platform with the explainability feature enabled, and provide this model to
+the tool through the standard `set_ai_platform_model` method, then attribution values will
+automatically be generated and visualized by the tool with no additional setup needed.
+If you wish to view attribution values for a different model setup, this can be accomplished through
+use of the custom prediction function.
+
+As described in the `set_custom_predict_fn` documentation in
+[WitConfigBuilder](https://github.com/tensorflow/tensorboard/blob/master/tensorboard/plugins/interactive_inference/witwidget/notebook/visualization.py), this method must return a list of the same size as the number of examples
+provided to it, with each list entry representing the prediction-time information for that example.
+In the case of a standard model with no attribution information, the list entry is just a number
+(in the case of a regression model), or a list of class probabilities (in the case of a classification model).
+
+But, if there is attribution or other prediction-time information that can be provided, then the list entry
+can instead be a dictionary, with the standard model prediction output under the `predictions` key. Attribution
+information can be returned under the `attributions` key and any other supplimental information under its own
+descriptive key. The exact format of the attributions and other supplimental information can be found in the code
+documentation linked above.
+
+If attribution values are provided to the What-If Tool, they can be used in a number of ways. First, when selecting
+a datapoint in the Datapoint Editor tab, the attribution values are displayed next to each feature value and the
+features can be ordered by their attribution strength, instead of alphabetically. Additionally, the feature values
+are colored by their attribution values for quick interpretation of attribution strengths.
+
+Beyond displaying the attribution values for the selected datapoint, the attribution values for each feature can be
+used in the tool in the same ways as any other feature of the datapoints. They can be used selected in the
+datapoints visualization controls to use those values to create custom scatter plots and histograms.
+For example, you can create a scatterplot showing the relationship between the attribution value of two different features,
+with the datapoints colored by the predicted result from the model.
+They can also be used in the Performance tab as a way to slice a dataset for comparing performance statistics of different slices.
+For example, you can quickly compare the aggregate performance of a model on datapoints with low attribution of a
+specified feature, against the datapoints with high attribution of that feature.
+
+Any other supplimental information returned from a custom prediction function can also be used in the same way, driving
```

Comment:
Any other supplemental information returned from a custom prediction function will appear in the tool as a feature named after its key in the dictionary. They can be used in the same way, ...

Reply:
done